river/Output.zig

```diff
@@ -55,7 +55,7 @@ const State = struct {
 };
 
 wlr_output: *wlr.Output,
-damage: *wlr.OutputDamage,
+damage: ?*wlr.OutputDamage,
```

Comment:
Making this optional and setting it to `null` in `handleDamageDestroy()` is nice even though we now need `.?` as we're now guaranteed to get a panic in safe build modes if we try to use it after it has been destroyed.